adapter/denier/BUILD

```diff
@@ -12,7 +12,7 @@ go_library(
         "//template/checknothing:go_default_library",
         "//template/listentry:go_default_library",
         "//template/quota:go_default_library",
-        "@com_github_googleapis_googleapis//:google/rpc",
+        "@io_istio_api//google/rpc:go_default_library",
```

Comment:
It looks very weird to me having google/rpc proto under istio.io/api. Is there any way to avoid doing this, like using `google.golang.org/genproto` packages?

Reply:
Unfortunately, gogo library of istio api is not build-compatible with genproto version. This would not be a problem if you use go library of istio api.